src/extras-openmoji/climate-environment/E2CE.svg

```diff
@@ -0,0 +1,24 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
wouldn't have the inner stroke of the meat connecting with the fork

Reply:
![2020-02-27 at 12 01](https://user-images.githubusercontent.com/480224/75438805-e740d000-5958-11ea-8163-ffeae6f03aea.jpg)